pkg/xds/bootstrap/testdata/generator.metrics-config.kubernetes.golden.yaml

```diff
@@ -37,6 +37,7 @@ node:
   id: mesh.name.namespace
   metadata:
     accessLogSocketPath: /tmp/kuma-al-name.namespace-mesh.sock
+    dataplane.admin.address: 127.0.0.1
```

Comment:
can we add a test case with ipv6?

Reply:
added tests, but not for bootstrap code because there is no ipv6 logic. I added tests for the `admin_proxy_generator.go`